internal/api/grpc/admin/import.go

```diff
@@ -726,68 +735,67 @@ func (s *Server) importData(ctx context.Context, orgs []*admin_pb.DataOrg) (*adm
 		}
 	}
 
-	if success != nil && success.Orgs != nil {
-		for _, org := range orgs {
-			var successOrg *admin_pb.ImportDataSuccessOrg
-			for _, oldOrd := range success.Orgs {
-				if org.OrgId == oldOrd.OrgId {
-					successOrg = oldOrd
-				}
-			}
-			if successOrg == nil {
-				continue
+	for _, org := range orgs {
```

Comment:
we could also loop over the success.orgs so that the check if org was success is unnecessary, but as a whole the change is correct

Reply:
this would only change the lines 739-746 because we need the org for the data to add. e.g. see line 749

Reply:
yeah would mean to change the item we are looping over